home.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:io';
+import 'Login.dart';
+import 'Sign-up.dart';

```

Comment:
at this stage we don't have any sign up or login so these shouldn't be part of this PR, please move each one to the associated PR.